packages/web/src/components/prompts.ts

```diff
@@ -21,25 +21,33 @@ function stripAnsiEscapes(str: string): string {
 
 export function fixTestPrompt(error: string, diff?: string, pageSnapshot?: string) {
   const promptParts = [
-    'This test failed, suggest how to fix it. Please be correct, concise and keep Playwright best practices in mind.',
+    `My Playwright test failed. What's going wrong?`,
+    `Please give me a suggestion how to fix it, and then explain what went wrong. Be very concise and apply Playwright best practices.`,
```

Comment:
I'm not sure about the `explain` part, personally I'd like to see a fix which I could apply ASAP and only have an explanation on demand.

Reply:
Yeah, we'll have to find a way of making this customizable!